gc/stats/ConcurrentPhaseStatsBase.hpp

```diff
@@ -47,6 +47,7 @@ class MM_ConcurrentPhaseStatsBase : public MM_Base
 	uintptr_t _bytesScanned;	/**< The number of bytes a given concurrent task did scan before it terminated (can be lower than _scanTargetInBytes if the termination was asynchronously requested) */
 	bool _terminationWasRequested;	/**< todo: remove after downstream projects start using _terminationRequestType */
 	uintptr_t _concurrentCycleType;	/**< The "type" of the corresponding cycle */
+	uintptr_t _concurrentMarkStartTime; /**< cpu start time of a concurrent mark increment */
```

Comment:
@RSalman as part of recent VGC work, do you remember is this info is already available elsewhere?

Reply:
Start/End times were added in `MM_ConcurrentMarkPhaseStats` for Standard CM. This was required because we were adding `concurrent-end` stanza and we weren't tracking TS/duration.

I recall this not being an issue for Balanced there were existing TS being tracked (for existing VGC CM stanzas)

Reply:
```C++
MM_MarkVLHGCStats *markStats = &static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats._markStats;
bool deltaTimeSuccess = getTimeDeltaInMicroSeconds(&duration, markStats->_startTime, markStats->_endTime);

handleGCOPOuterStanzaStart(env, "mark increment", stats->_cycleID, duration, deltaTimeSuccess);
```

Reply:
>Start/End times were added in MM_ConcurrentMarkPhaseStats for Standard CM.

btw these are being moved to base class  (CS is aligning to use them as well): https://github.com/eclipse/omr/pull/6068

Reply:
OK, thanks. 
So, @cedrichansen check if you could rely on _vlhgcIncrementStats._markStats->_startTime instead (potential problems are that it may be initialized too early or too late).

Reply:
These two fields are being measured differently.
`_concurrentMarkStartTime ` is being measured by looking at cpu process user time + system time, to account for threads stalling. This method is useful for properly weighing the true cost of GMP.

@amicic Do you want me to give the new field (`markStats->_startTime`) a try anyways?

Reply:
ok, I see. let's rename it. 'start' is ok to keep around, but you need something extra in the name like 'CPU' or 'busy' or 'process'  ...StartTime.
and comment properly

Reply:
The field has been renamed to `_concurrentMarkProcessStartTime `